packages/xo-web/src/xo-app/dashboard/health/index.js

```diff
@@ -174,6 +178,33 @@ const ORPHANED_VDI_COLUMNS = [
   },
 ]
 
+const AttachedVdisActions = [
+  {
+    handler: disconnectVbds,
+    icon: 'disconnect',
+    individualHandler: disconnectVbd,
+    individualLabel: _('vbdDisconnect'),
+    label: _('vbdsDisconnect'),
+    level: 'danger',
+  },
+  {
+    handler: deleteVbds,
+    individualHandler: deleteVbd,
```

Comment:
This should look like:
`({ vbd }) => deleteVbd(vbd)`

Reply:
Same for the "disconnect" action.

Reply:
we don't need it because the collection contains `ids` of `vbds`

Reply:
True.
(Even if `deleteVbd` could require an VBD object in the future. But that's another problem.)